src/main/scala/gui/EventsScene.scala

```diff
@@ -0,0 +1,105 @@
+package gui
+
+import java.net.URI
+import java.awt.Desktop
+
+import scalafx.Includes._
+import events.Event
+import scalafx.collections.ObservableBuffer
+import scalafx.scene.Scene
+import scalafx.scene.control.Alert.AlertType
+import scalafx.scene.control.{ Label, TextField}
+import scalafx.stage.Stage
+import scraping.{LiveNationScraper, Monitor, SearchEngine}
+
+class EventsScene(private val sceneWidth: Double, private val sceneHeight: Double, private val stage: Stage
+                  , private val scraper: LiveNationScraper, private val monitor: Monitor) extends Scene(sceneWidth, sceneHeight) {
+
+  private val offYTopLabel = 20
+  private val offYSearch = 50
+  private val offYList = 80
+  private val offYDown = 560
+  private val textFieldWidth = 200
+  private val offX = 20
+  private val buttonWidth = 150
+  private val listWidth = 560
+
+  val label = new Label("Search for concert:")
+  label.layoutX = offX
+  label.layoutY = offYTopLabel
+
+  val textField = new TextField
+  textField.layoutX = offX
+  textField.layoutY = offYSearch
+  textField.promptText = "concert/star/band name"
+  textField.prefWidth = textFieldWidth
+
+  val searchButton = new EventsButton("Search", offX + textFieldWidth, offYSearch)
```

Comment:
This should probably be split to methods, like prepareButtons, addActionHandlers or something like that